### PR TITLE
Update README URLs based on HTTP redirects

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein
### GitHub Corrected URLs

| Was | Now |
| --- | --- |
| https://github.com/edhelas/movim | https://github.com/movim/movim |
| https://github.com/embolalia/willie/wiki/Commands | https://github.com/sopel-irc/sopel/wiki/Commands |
| https://github.com/libreboard/libreboard | https://github.com/wekan/wekan |
### HTTPS Corrected URLs

| Was | Now |
| --- | --- |
| http://burnsoftware.wordpress.com/dayjournal/ | https://burnsoftware.wordpress.com/dayjournal/ |
| http://burnsoftware.wordpress.com/daytasks/ | https://burnsoftware.wordpress.com/daytasks/ |
| http://en.wikipedia.org/wiki/Kanban_board | https://en.wikipedia.org/wiki/Kanban_board |
| http://en.wikipedia.org/wiki/RoundCube | https://en.wikipedia.org/wiki/RoundCube |
| http://gogs.io/ | https://gogs.io/ |
| http://gravit.io/ | https://gravit.io/ |
| http://owncloud.org/ | https://owncloud.org/ |
| http://prism-break.org/en/ | https://prism-break.org/en/ |
| http://syncthing.net/ | https://syncthing.net/ |
| http://taskboard.matthewross.me/ | https://taskboard.matthewross.me/ |
| http://wordpress.org/ | https://wordpress.org/ |
| http://www.youtube.com/playlist?list=PL97A379365B6C19BD | https://www.youtube.com/playlist?list=PL97A379365B6C19BD |
### Other Corrected URLs

| Was | Now |
| --- | --- |
| http://cozy.io/en/hack/cookbooks/components.html#applications | https://docs.cozy.io/en/hack/cookbooks/components.html#applications |
| http://cozy.io/en/hack/getting-started/ | https://docs.cozy.io/en/hack/getting-started/ |
| http://cozy.io/en/host/install/ | https://docs.cozy.io/en/host/install/ |
| http://meteor.com/ | https://www.meteor.com/ |
| http://roundcube.net/screens | http://roundcube.net/screens/ |
| http://seafile.com/en/home/ | https://www.seafile.com/en/home/ |
| http://smallpicture.com/outlinerHowto.html | http://fargo.io/docs/outlinerHowto.html |
| http://status.net/ | https://e14n.com/ |
| http://thinkupapp.com/ | https://www.thinkup.com/ |
| http://tonido.com/ | http://www.tonido.com/ |
| http://willie.dftba.net/ | https://sopel.chat/ |
| http://www.a2hosting.com/services/vps-hosting/ | https://www.a2hosting.com/vps-hosting |
| http://www.lowendbox.com | https://lowendbox.com/ |
| http://www.softsyshosting.com/windows-vps.aspx | https://www.softsyshosting.com/windows-vps/ |
| http://www.vmware.com/products/zimbra/overview.html | http://www.vmware.com/products/zimbra.html |
| http://www.zimbra.com/downloads/os-downloads.html | https://www.zimbra.com/downloads/zimbra-collaboration-open-source/ |
| https://pyd.io/ | https://pydio.com/ |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,16 +148,16 @@
 	- Sync your contacts, calendars and files between your devices and your personal server
 	- [Demo](https://demo.cozycloud.cc/)
 	- [Source Code](https://github.com/cozy/cozy-setup)
-	- Runs on Linux, available for VirtualBox, Docker and DigitalOcean. [Installations](http://cozy.io/en/host/install/).
+	- Runs on Linux, available for VirtualBox, Docker and DigitalOcean. [Installations](https://docs.cozy.io/en/host/install/).
 	- Looks really good. Built using Node.js
-	- [Built custom apps](http://cozy.io/en/hack/getting-started/)
-	- [Already existing custom apps](http://cozy.io/en/hack/cookbooks/components.html#applications)
-- [Pydio](https://pyd.io/)
+	- [Built custom apps](https://docs.cozy.io/en/hack/getting-started/)
```
Comment (AmrEldib) [inline README.md:153]: Should be https://dev.cozy.io/#getting-started

File: README.md [hunk 2]
```diff
@@ -220,13 +220,13 @@
 - [Shrinkr](http://shrinkr.codeplex.com/) 	
 
 ## Social Network
-- [ThinkUp](http://thinkupapp.com/) to collect Tweets and Facebook data. 
+- [ThinkUp](https://www.thinkup.com/) to collect Tweets and Facebook data. 
 - [Diaspora](http://diasporaproject.org/) Social Network
 - Twitter-like Social Media Network
-	- [StatusNet](http://status.net/) 
+	- [StatusNet](https://e14n.com/)
```
Comment (AmrEldib) [inline README.md:226]: Update to GnuSocial https://gnu.io/social/ instead of StatusNet

**Comment by ReadmeCritic** [comment]
@AmrEldib oops deleted the fork.. if the edits are minor, do you mind making them yourself after merge?

File: README.md [hunk 3]
```diff
@@ -270,8 +270,8 @@
 	- [Features](https://about.gitlab.com/features/)
 
 ## Personal Assistant
-- [Willie](http://willie.dftba.net/) is a simple, lightweight, open source, easy-to-use IRC utility bot, written in Python. It can leave notes for people, give you reminders, check RSS feeds.
-	- [List of Commands](https://github.com/embolalia/willie/wiki/Commands).  
+- [Willie](https://sopel.chat/) is a simple, lightweight, open source, easy-to-use IRC utility bot, written in Python. It can leave notes for people, give you reminders, check RSS feeds.
```
Comment (AmrEldib) [inline README.md:273]: Update the name to `sopel`

**Comment by AmrEldib** [comment]
@ReadmeCritic yes, I will. I was just making notes to make additional updates after merging
Thank you for your help!

**Pull request merged**